docs/config/mouse.md

```diff
@@ -70,6 +70,13 @@ return {
       mods="CTRL",
       action="OpenLinkAtMouseCursor",
     },
+    -- NOTE: if you only bind the 'Up' event in wezterm, the 'Down' event will
```

Comment:
I feel like this would be better off outside of the code comment and be part of the main prose below!

Reply:
Maybe yes, but I wanted to put it next to the Ctrl-Click binding to ensure readers see it.
Or I put the explanation below, and only a small note here, saying to read more below?

Reply:
I think having the main explanation outside the code seems best, but yeah, have a short comment in the example pointing the reader to the rest of the docs.